apps/rig_tests/test/avro/avro_test.exs

```diff
@@ -59,16 +57,15 @@ defmodule RigTests.Avro.AvroTest do
       })
 
     kafka_config = kafka_config()
+    :timer.sleep(5_000)
```

Comment:
Why is this (and the other `sleep`s below) needed?

Reply:
From my observations, from time to time clients need some time to be in ready state and since we miss proper ready function for that, I chose this -- didn't find better and stable approach.

Reply:
aah _that_ issue...